tests/test_reclice.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+from mpi4py import MPI
+from httomo.data.hdf._utils.chunk import get_data_shape
+from httomo.data.hdf._utils.load import get_slice_list_from_preview
+
+from httomo.data.hdf._utils.reslice import reslice
+import pytest
+
+
+@pytest.mark.parametrize(
+    "current_slice_dim, next_slice_dim",
+    [(1, 2), (2, 1), (1, 3), (2, 3), (3, 1), (3, 2)],
+    ids=[
+        "sino2proj",
+        "proj2sino",
+        "sino2third",
+        "proj2third",
+        "third2sino",
+        "third2proj",
+    ],
```

Comment:
No matter how many times I've gone over this sort of thing in the past I still doubt myself, but I _think_ that some of these labels may be the wrong way around? Ie, `(1, 2)` means `current_slice_dim=1, next_slice_dim=2` doesn't it, which would be going from projections to sinograms (ie, `"proj2sino"`) rather than `"sino2proj"`?

Reply:
Sorry, maybe a more general and useful thing to say is:
- if `current_slice_dim=1`, then we have projections
- if `current_slice_dim=2`, then we have sinograms
- if `current_slice_dim=3`, then we have "third/something unnamed"

Reply:
you're right - just to get this straight:
- 1 => projections
- 2 => sinogram
- 3 => no specific name - I called it `third`

Reply:
Yep, that's right :) And `third` is as good a name as any, straight away I know what it means!

Reply:
done